CMakeLists.txt

```diff
@@ -98,10 +103,35 @@ if (MASTER_PROJECT)
 endif()
 
 # Export
-include(GNUInstallDirs)
-
 install(TARGETS oaknut EXPORT oaknutTargets)
 install(EXPORT oaknutTargets
     NAMESPACE merry::
     DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/oaknut"
 )
+
+option(OAKNUT_PACKAGE_MODE "Install addition CMake files for standalone oaknut install" OFF)
```

Comment:
I do not think wrapping this in `OAKNUT_PACKAGE_MODE` is necessary. One can externally control installation already with standard CMake variables. e.g. by using the `EXCLUDE_FROM_ALL` property.